### DISCUSSION TITLE
How do I resolve this service resolution error with Dry IoC

### DISCUSSION THREAD

**Discussion by steveodai** [PrismLibrary/Prism#2667] [Xamarin.Forms]
I use DryIoC with my Xamarin Forms Prism MVVM application and I have a service resolution error. Please see https://stackoverflow.com/questions/71130859/xamarin-prism-dry-ioc-service-resolution-error?noredirect=1#comment125756113_71130859 for more information.

**Comment by brianlagunas** [comment]
Without a reproduction app, no one can help you.
